src/main.js

```diff
@@ -0,0 +1,502 @@
+"use strict";
+const TASK_COUNT = 3;
+
+// ----------------- Шаблоны -------------------------------------
+
+//  кнопки меню
+const createHeaderBtnWrapTemplate = () => {
+  return (`
+    <section class="control__btn-wrap">
+      <input
+        type="radio"
+        name="control"
+        id="control__new-task"
+        class="control__input visually-hidden"
+      />
+      <label for="control__new-task" class="control__label control__label--new-task"
+        >+ ADD NEW TASK</label
+      >
+      <input
+        type="radio"
+        name="control"
+        id="control__task"
+        class="control__input visually-hidden"
+        checked
+      />
+      <label for="control__task" class="control__label">TASKS</label>
+      <input
+        type="radio"
+        name="control"
+        id="control__statistic"
+        class="control__input visually-hidden"
+      />
+      <label for="control__statistic" class="control__label"
+        >STATISTICS</label
+      >
+    </section>
+  `);
+};
+
+//  кнопки фильтры
+const createHeaderMainFilterTemplate = () => {
+  return (`
```

Comment:
```suggestion
  return (
    `
```
В данном случае немного избыточно и скобки и ` на одной строке
![carbon (11)](https://user-images.githubusercontent.com/25539007/69299579-eeea2d80-0c21-11ea-982c-dc3bbeca87a7.png)